src/pages/Feedback.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+
+export default function Feedback({ feedbackSubmit, elementForFeedback }) {
+  function submitFeedback(event) {
+    event.preventDefault();
+    const form = event.target;
+    const { name, feedback } = form.elements;
+    feedbackSubmit(name.value, feedback.value, elementForFeedback);
+    console.log(name.value, feedback.value);
+    console.log(elementForFeedback);
+  }
+  return (
+    <Section>
+      <p>
+        <b> Deine Bewertung zum folgenden Artikel: </b>
+        <span>{elementForFeedback.title}</span>, <b> gekauft am </b> 
+        <span>{elementForFeedback.date}</span>. <b> Gesamtmenge: </b>{" "}
+        <span>{elementForFeedback.amount}</span>
+        <b> Paar. </b> <b> Preis: </b> <span>{elementForFeedback.price}</span> 
+        <b> € je Paar. </b>
+      </p>
+      <form onSubmit={submitFeedback}>
+        <fieldset>
+          <legend>Deine Bewertung</legend>
+          <label>Username</label>
```

Comment:
The labels are not properly connected to the inputs. Please use htmlFor or the input as a child of the label.

Reply:
thank you, i changed it!